.kitchen.yml

```diff
@@ -150,10 +150,11 @@ suites:
           backend: ssh
           controls:
             - real-time-enforcer-host
-          hosts_output: forseti-rt-enforcer-vm-public-ip
+          hosts_output: forseti-rt-enforcer-vm-ip
           user: ubuntu
           key_files:
             - test/fixtures/real_time_enforcer/sshkey
+          proxy_command: ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -o IdentitiesOnly=yes -o LogLevel=ERROR -o ForwardAgent=no -i test/fixtures/bastion/tls_private_key ubuntu@<%= `terraform output -state test/fixtures/real_time_enforcer/terraform.tfstate.d/kitchen-terraform-real-time-enforcer-local/terraform.tfstate bastion_host`.strip %> -p 22 -W %h:%p
```

Comment:
The `bastion_host` output must be added to `test/fixtures/real_time_enforcer/outputs.tf`

Reply:
Done